src/qibolab/platforms/multiqubit.py

```diff
@@ -28,10 +92,31 @@ def execute_pulse_sequence(self, sequence: PulseSequence, nshots=None):
         # Process Pulse Sequence. Assign pulses to instruments and generate waveforms & program
         instrument_pulses = {}
         roles = {}
+
+        changed = {}
         for name in self.instruments:
             roles[name] = self.settings["instruments"][name]["roles"]
             if "control" in roles[name] or "readout" in roles[name]:
                 instrument_pulses[name] = sequence.get_channel_pulses(*self.instruments[name].channels)
+                # Change pulses frequency to if and correct lo accordingly (before was done in qibolab)
+
+                if "readout" in roles[name]:
+                    for pulse in instrument_pulses[name]:
+                        if abs(pulse.frequency) > 300e6:
```

Comment:
These constants here are weird (I mean `300e6`). If they are meaningful, give them a meaningful name and document, making a top-level constant, such that it can be accessed also from outside and changed consistently

```py
ALIENS_FREQ = 300e6
"""The most relevant threshold freq to discriminate different types of aliens."""
```